src/test/config.sol

```diff
@@ -98,38 +98,38 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0),        // populate with deployed spell if deployed
```

Comment:
If you can easily rollback these spacing changes would be appreciated so we can try to reduce to the minimum the code changes.

Reply:
I'll do that, however wouldn't it be better if we "fixed" this?

All tools that I know that help automating this (both on Vim or VSCode) will produce the same result by default.

Otherwise we're stuck manually adjusting spaces forever.

Reply:
We use to do PRs for styling fixes, I'd say we discuss what is the proper way through one of them.
It is true that sometimes we also fix styling stuff during new executive PRs, but I'd say that as this is all new process we try to keep restricted to the minimum that needs to change.